dvc/repo/experiments/utils.py

```diff
@@ -342,3 +343,63 @@ def to_studio_params(dvc_params):
             result[file_name] = file_data.get("data", {})
 
     return result
+
+
+def _describe(
```

Comment:
```suggestion
def describe(
```

Since this is now a public `experiments.utils` method we should drop the `_` prefix

Reply:
Makes sense, done.